README.md

```diff
@@ -19,3 +19,17 @@ steps:
         ## My heading\n
         - Markdown is supported\n
 ```
+
+## Inputs:
+
+### Title
+The title of the issue to create. Required.
+`title: Title of my issue`
+
+### Description
+The description of the issue to create. Required.
+`composer_install: true`
```

Comment:
I'm guessing that was a copy/paste mistake? 😅 Should give an example of `description`, not `composer_install`

Reply:
Done